src/Bicep.Core/Analyzers/Linter/Rules/SimplifyInterpolationRule.cs

```diff
@@ -44,6 +44,12 @@ public Visitor(Dictionary<TextSpan, CodeFix> spanFixes, SemanticModel model)
                 this.model = model;
             }
 
+            public override void VisitMetadataDeclarationSyntax(MetadataDeclarationSyntax syntax)
+            {
+                AddCodeFixIfSingleInterpolatedString(syntax.Value);
+                base.VisitMetadataDeclarationSyntax(syntax);
+            }
```

Comment:
This probably isn't necessary if we're going to restrict the functionality to not allow expressions.